people.md

```diff
@@ -5,9 +5,34 @@ permalink: /people/
 ---
 {%- for i in site.data.people -%}
 {%- if i.members.size > 0 -%}
-# {{ i.title }}
-{%- for j in i.members %}
-[{{ j.name }}]({{ j.link }}), {{ j.affiliation }}
-{% endfor -%}
+## {{ i.title }}
+<table style="table-layout:fixed; width:100%; border: none;">
+<tbody style="background: none;">
+<tr style="background: none;">
+{% assign k = 0 -%}
+{%- for j in i.members -%}
+{%- if j.img -%}
+{%- capture img -%}{{ j.img }}{%- endcapture -%}
+{%- else -%}
+{%- capture img -%}/assets/images/people/null.jpg{%- endcapture -%}
+{%- endif -%}
+{%- capture name -%}{{ j.name }}{%- endcapture -%}
+{%- capture link -%}{{ j.link }}{%- endcapture -%}
+{%- capture affiliation -%}{{ j.affiliation }}{%- endcapture -%}
+{%- include person.html img=img name=name link=link affiliation=affiliation -%}
+{%- assign k = k | plus:1 -%}
+{%- if k == 4 -%}
+</tr>
+{% unless forloop.last %}
+<tr style="background: none;">
+{% endunless %}
+{%- assign k = 0 -%}
+{%- endif -%}
+{%- endfor -%}
+{%- if k > 0 -%}
+</tr>
+{% endif -%}
+</tbody>
+</table>
```

Comment:
I suspect there's a better way to do this using a [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox) or maybe a [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Relationship_of_Flexbox_to_Other_Layout_Methods#Flexbox_and_Grid_Layout) with a media query to set the number of columns based on browser size. But this table seems strictly better than what we had.

Reply:
TIL how to use a grid :)